opentracing-cdi/pom.xml

```diff
@@ -22,8 +22,21 @@
       <artifactId>opentracing-api</artifactId>
     </dependency>
     <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-tracerresolver</artifactId>
+    </dependency>
+    <dependency>
       <groupId>io.opentracing</groupId>
       <artifactId>opentracing-util</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-ejb</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-web-servlet-filter</artifactId>
```

Comment:
True, but if we want the servlet integration, we need to create a producer that gets data from the request context, and this requires this dependency. Should I remove the SpanContext producer?